### ISSUE TITLE
Error re: vbmc_runner pickling on starting vbmcd

### ISSUE THREAD

**Issue by bpengu1n** [kurokobo/virtualbmc-for-vsphere#2]
Environment:
   macOS 11.2.3
   Python 3.9.0 (installed via homebrew)
   Latest pip/pypi index+packages

I followed the documentation in installing the package, and on attempting to start vbmcd, I get the following error:
```bash
~ » vbmcd --foreground
2021-03-21 00:17:20,414.414 65337 ERROR VirtualBMC [-] Can't pickle local object 'VirtualBMCManager._sync_vbmc_states.<locals>.vbmc_runner'
```

vbmcd does not show in process list or netstat, and vbmc reports an error attempting to connect, so definitely not running. Only other steps I took prior were vbmc add .... and vbmc start ....

Noticed that if I start from scratch, vbmcd does start initially, but dies after adding my server and calling vbmc start ....

config contents (w/ creds + IPs redacted):
```
[VirtualBMC]
username = [BMC-USERNAME]
password = [BMC-PASSWORD]
address = ::
port = 6230
fakemac = 02:00:00:4e:70:e5
vm_name = fangorn-vesxi01
viserver = [IP]
viserver_username = [VC-USERNAME]
viserver_password = [VC-PASSWORD]
active = True
```

**Comment by kurokobo** [comment]
Thanks for the report! I hadn't tried to get it to work on macOS.

It seems to be something around multiprocessing. I'll look into it.

**Comment by kurokobo** [comment]
Sorry for late response, I've released new version `0.0.7` including your PR 😃 
Thanks for your contribution!!
